src/main/java/roboy/newDialog/NewDialogSystem.java

```diff
@@ -0,0 +1,63 @@
+package roboy.newDialog;
+
+
+import roboy.dialog.Config;
+import roboy.dialog.action.Action;
+import roboy.io.*;
+import roboy.linguistics.sentenceanalysis.*;
+import roboy.talk.Verbalizer;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static roboy.dialog.Config.ConfigurationProfile.NOROS;
+
+/**
+ * Temporary class to test new state based personality.
+ * Will be be extended and might replace the old DialogSystem in the future.
+ */
+public class NewDialogSystem {
+
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+
+        new Config(NOROS);
+
+
+        MultiInputDevice multiIn = new MultiInputDevice(new CommandLineInput());
+        MultiOutputDevice multiOut = new MultiOutputDevice(new CommandLineOutput());
+        List<Analyzer> analyzers = new ArrayList<>();
+        analyzers.add(new Preprocessor());
+        analyzers.add(new SimpleTokenizer());
+
+        StateBasedPersonality personality = new StateBasedPersonality(new Verbalizer());
+        personality.loadFromFile(new File("resources/personalityFiles/ExamplePersonality.json"));
```

Comment:
file path -> config yaml

Reply:
This one: https://github.com/Roboy/roboy_dialog/blob/devel/config.properties ?